proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,133 @@
+# Cosmos Stargate Hub Upgrade Proposal 2: Chevrons Locked. Time to Upgrade
+
+![Stargate Upgrade banner graphic](https://siasky.net/AADU5rg0GvapbrICLUk7SxZeMDrKo5Q8oLrpOwl71wXhnQ)
+
+## by Iqlusion
+
+Jan 11, 2021
+
+## Key Results
+
+------------
+If passed, this governance proposal would commit the Cosmos Hub to halting the `cosmoshub-3` at 06:00 UTC on Jan 28th exporting the state and starting `cosmoshub-4` based on gaia 3.0.
+
+## Context
+
+------------
+
+In proposal 27, Iqlusion proposed a comprehensive process to translate the unprecented surface area of this upgrade. We called this process Stargate.
+
+Iqlusion is pleased to report that the Stargate Process has been successfully executed. We believe the chevrons are unlocked and that Hub can safely step through the Stargate.
+
+After the success of the community approval of the first Stargate Upgrade proposal, we now follow-up with the second Stargate proposal for the Cosmos Hub that will outline the results of the first proposal and the process for upgrading the Hub.
+
+## The Stargate Plan
+
+------------
+The purpose of the Stargate effort was to ensure that Cosmos can fulfill the vision of an Internet of Blockchains in 2021 while mitigating the risks outlined above. It was also an attempt to create a process for complex future on-chain upgrades on the Cosmos Hub. That included the following steps:
```

Comment:
> the risks outlined above

Personally, I know what these risks are, but they aren't outlined above in this proposal